### ISSUE TITLE
Additional events not accessible via keyboard, 508 issue

### ISSUE THREAD

**Issue by nnarvekar** [jquense/react-big-calendar#2045]
#### What's the current behavior?
I have the calendar set to the month view. When it displays the "+2 more" link, a popup with the events is displayed. But this is not accessible/read by the screen reader JAWS.
![image](https://user-images.githubusercontent.com/23129870/137146201-0d6b17b4-5e8c-4546-9e36-5f1589674b7a.png)
![image](https://user-images.githubusercontent.com/23129870/137146553-d2fc5df3-598e-4610-9294-454f95c7ecc8.png)

#### What's the expected behavior?
The screen reader should be able to read the events displayed when the "+2 more" link is selected

**Comment by cutterbl** [comment]
We probably need ARIA attributes everywhere within RBC. Not being overly familiar myself, what would proper attributes be here? If I knew what they should be I could submit a PR (or you could).

**Comment by nnarvekar** [comment]
> We probably need ARIA attributes everywhere within RBC. Not being overly familiar myself, what would proper attributes be here? If I knew what they should be I could submit a PR (or you could).

@cutterbl I do not know the ARIA attributes to use
